.github/workflows/generate-build-time-stats.yml

```diff
@@ -0,0 +1,52 @@
+name: Generate build time statistics 
```

Comment:
Do we need this workflow? We have added it in master. I think we can remove this